extensions/stock-api-fastapi/README.md

```diff
@@ -0,0 +1,16 @@
+# FastAPI Stock Pricing Service
+
+## About this example
+
+An API allows you to turn your models into production services that other tools and teams can use. APIs are a great way for software engineering teams to use your models without translating them into different languages.
```

Comment:
I'm gonna have the exact same comments about these sentences :)

```suggestion
APIs allow you to turn your models into production services that other tools and teams can use without translating them into different languages.
```

Reply:
Yeah the original descriptions are leaning a lot on the names to differentiate them the API jump starts from each other. 

I do think @cgraham-rs has a point about the descriptions already being provided for us. We can always make an additional pass once we get these in.